workers/generic-worker/multiuser_linux.go

```diff
@@ -19,6 +19,7 @@ func platformFeatures() []Feature {
 		&InteractiveFeature{},
 		&LoopbackAudioFeature{},
 		&LoopbackVideoFeature{},
+		&RunTaskAsCurrentUserFeature{},
```

Comment:
I suspect having this is up higher in the list might be safer, if it changes `pd` and the other features depend on `taskContext.pd`.